files-community/Files
# RichCommand: Refresh items

**Resolved / Related Issues**
- Implements rich command for refresh

**Validation**
How did you test these changes?
   1. Open a folder and click the refresh button in the address bar
   2. Use the `ctrl + r` keyboard shortcut
   3. Right click in a blank space and verify the refresh option works

**Screenshots (optional)**
Add screenshots here.
